bugrap/src/main/java/org/vaadin/bugrap/ui/ReportsView.java

```diff
@@ -140,4 +156,15 @@ private void revertChanges() {
 		model.resetReportForEdit();
 		binder.setBean(model.getReportForEdit());
 	}
+	
+	private void openReportDetail() {
+		Window window = new Window();
+		window.setCaption(String.format("%s > %s", model.getReportForEdit().getProject().getName(), model.getReportForEdit().getVersion().getVersion()));
+		window.setSizeFull();
+		
+		ReportDetailView view = new ReportDetailView(model);
+		window.setContent(view);
+		
+		getUI().addWindow(window);
```

Comment:
Remember using the specific Vaadin component here.

Reply:
Do you mean BrowserWindowOpener?

Reply:
Yes.